src/lib/themes.js

```diff
@@ -116,17 +120,20 @@ export const normalizeThemeBackground = background =>
   bgImages.keys().includes(background) ? background : null;
 
 // Utility to ensure normal & consistent colors
-export const normalizeThemeColor = (data, defaultColor) =>
-  makeTinycolor(data || defaultColor).toRgb();
+export const normalizeThemeColor = (name, data, defaultColor) => {
```

Comment:
This is getting called independently of normalizeThemeColors in the redux store causing alpha values to leak. I moved the alpha check into this function to resolve